tests/unit/splinter_test.c

```diff
@@ -428,10 +429,11 @@ CTEST2(splinter, test_lookups)
 
    trunk_shadow shadow;
    trunk_shadow_init(&shadow);
+   uint64 num_inserts = 0;
 
    // FALSE : No need to do verification-after-inserts, as that functionality
    // has been tested earlier in test_inserts() case.
-   uint64 num_inserts = splinter_do_inserts(data, spl, FALSE, &shadow);
+   num_inserts = splinter_do_inserts(data, spl, FALSE, &shadow);
```

Comment:
Why this change? I generally strongly prefer defining variables where they are used.

Reply:
Don't recall why I edited this. Grubby fingers. Reverted.